docs/pages/upgrading/self-hosted-automatic-agent-updates.mdx

```diff
@@ -23,216 +23,68 @@ present in their package repositories. As Teleport 14 won't be published to
-     bucket = aws_s3_bucket.version_server.id
-     policy = jsonencode({
-       Version = "2012-10-17"
-       Statement = [
-         {
-           Sid       = "GrantAnonymousReadPermissions"
-           Principal = "*"
-           Action = [
-             "s3:GetObject",
-           ]
-           Effect   = "Allow"
-           Resource = ["${aws_s3_bucket.version_server.arn}/*"]
-         },
-       ]
-     })
-   }
-   
-   resource "aws_s3_bucket_public_access_block" "version_server" {
-     bucket = aws_s3_bucket.version_server.id
-   
-     block_public_acls       = false
-     block_public_policy     = false
-     ignore_public_acls      = false
-     restrict_public_buckets = false
-   }
-   ```
-   </TabItem>
-   <TabItem label="Google Cloud Storage">
-
-   In the configuration below, replace `PROJECT` and `REGION` with the Google
-   Cloud project and region where you will deploy the version server:
-   
-   ```hcl
-   terraform {
-     required_providers {
-       google = {
-         source  = "hashicorp/google"
-         version = "~> 5.0"
-       }
-     }
-   }
-
-   provider "google" {
-     project     = "PROJECT"
-     region      = "REGION"
-   }
-
-   resource "google_storage_bucket" "version_server" {
-     // Change the location as needed. See:
-     // https://cloud.google.com/storage/docs/locations
-     location = "US-EAST1"
-     // Replace this to ensure it is unique
-     name = "version-server"
-   }
-   
-   resource "google_storage_bucket_object" "version" {
-     name   = "current/version"
-     source = "${path.root}/current/version"
-     bucket = google_storage_bucket.version_server.name
-   }
-   
-   resource "google_storage_bucket_object" "critical" {
-     name   = "current/critical"
-     source = "${path.root}/current/critical"
-     bucket = google_storage_bucket.version_server.name
-   }
-
-   data "google_iam_policy" "viewer" {
-     binding {
-       role = "roles/storage.objectViewer"
-       members = [
-           "allUsers",
-       ] 
-     }
-   }
-   
-   resource "google_storage_bucket_iam_policy" "version_server" {
-     bucket = google_storage_bucket.version_server.name
-     policy_data = data.google_iam_policy.viewer.policy_data
-   }
-   ```
-   </TabItem>
-   </Tabs>
-
-1. Make your cloud provider credentials available to Terraform. The method to
-   use depends on your organization, e.g., pasting environment variables into
-   your terminal.
+<Details title="Changing which version is served by the version server">
+
+By default, the version server has a single `default` channel, serving the proxy
+version. If you want to override the default version or add other channels you
+can pass custom proxy configuration:
+
+```yaml
+proxy_service:
+  enabled: "yes"
+  automatic_upgrades_channels:
+    # Override the default version channel
+    default:
+      static_version: v14.2.1
+    # Define a new version channel with a static version
+    additional/channel/static:
```

Comment:
So any `/`-separated string key defines a route on the version server? Which path would the user expect to query here (is there a prefix for the path?)?

Reply:
I added a URL example and removed the slashes as it was confusing. We support slashes in the channel name, but they're not mandatory.